### DISCUSSION TITLE
Disable to update button for a certain package

### DISCUSSION THREAD

**Discussion by nothingness6** [LykosAI/StabilityMatrix#638] [Ideas]
I installed SD UI 1.7.0 for the old projects. The new version of Stability Matrix show the green update button for every package and I was close to click the update button for 1.7.0. 
Thus, I'd like to be able to control the update button. If you add the function that show/hide the update button for each package individually, it'll be very helpful.
Thanks!

**Comment by jffaust** [comment]
I'm wondering too if it's possible to disable the update/auto-update for a package. I saw the recent announcement for SD Forge (https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/801) and it would be preferable to stay at a specific version.
